userguide/sharing.rst

```diff
@@ -34,13 +34,15 @@ system to meet the requirements of the network.
 
 These types of shares and services are available:
 
-* :ref:`AFP <Apple (AFP) Shares>`: Apple File Protocol shares are
-  often used when the client computers all run macOS. Apple has
-  slowly shifted to preferring :ref:`SMB <Windows (SMB) Shares>` for
-  modern networks, although Time Machine still requires AFP.
+* :ref:`AFP <Apple (AFP) Shares>`: Apple Filing Protocol shares are
+  used when the client computers all run macOS. Apple has deprecated
+  AFP in favour of :ref:`SMB <Windows (SMB) Shares>` and using AFP
+  in modern networks is no longer recommanded (except for providing a
+  Time Machine, for which %brand% currently only supports AFP, not
+  yet SMB).
 
 * :ref:`Unix (NFS) <Unix (NFS) Shares>`: Network File System shares
-  are accessible from Mac OS X, Linux, BSD, and the professional and
+  are accessible from macOS, Linux, BSD, and the professional and
   enterprise versions (but not the home editions) of Windows. This can
   be are a good choice when the client computers do not all run the
```

Comment:
s/be are/be/

Reply:
Just because I did a find/replace of "Mac OS X" spelling doesn't mean I'm trying to find & correct every typo in this document...